.github/workflows/generate-semantic-conventions-pr.yaml

```diff
@@ -21,15 +21,14 @@ jobs:
         env:
           GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         run: |
-          # version in this repo are prefixed with v
           latest_version=v$(gh release view \
                                --repo open-telemetry/semantic-conventions \
                                --json tagName \
                                --jq .tagName \
                              | sed 's/^v//')
 
-          rc=$(find semconv -name $latest_version | grep .)
-          if $rc == 0; then
+          found=$(find semconv -name $latest_version)
+          if [[ $found =~ $latest_version ]]; then
             already_added=true
```

Comment:
Does it matter that this is using underscore and the output is defined with dash?

Reply:
It doesn't, you can see where the output is set on line 46:

```
echo "already-added=$already_added" >> $GITHUB_OUTPUT
```